app/workers/deletes.php

```diff
@@ -450,26 +457,41 @@ protected function deleteFunction(Document $document, Document $project): void
     {
         $projectId = $project->getId();
         $dbForProject = $this->getProjectDB($project);
+        $dbForConsole = $this->getConsoleDB();
         $functionId = $document->getId();
+        $functionInternalId = $document->getInternalId();
+
+        /**
+         * Delete routes
+         */
+        Console::info("Deleting routes for function " . $functionId);
```

Comment:
Let's remove this log. If we need any logs, it would be better to have a more generic strategy.

Reply:
Check the rest of the file, it's what we do everywhere. Should I remove it from there too?

Reply:
Keeping as is to stay consistent